modules/configuring-user-level-access.adoc

```diff
@@ -0,0 +1,47 @@
+// Module is included in the following assemblies:
+//
+// * installing-red-hat-openshift-gitops
+
+:_content-type: PROCEDURE
+[id="configuring-user-level-access_{context}"]
+= Configuring user level access
+
+[role="_abstract"]
+To manage and modify the user level access you need to configure the Argo CD RBAC configmap.
+
+.Procedure
+
+* Edit the `argocd-rbac-cm` configmap:
++
+[source,terminal]
+----
+$ oc edit cm argocd-rbac-cm -n <namespace>
+----
+.Output
```

Comment:
Please copy the output from any the argo cd instance. I can work with you to figure this out.